settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Neměl by být settings.json taky prázdnej protože je to jakoby open-source projekt?

Reply:
> Neměl by být settings.json taky prázdnej protože je to jakoby open-source projekt?

Discord bot bych tipoval je soukromy projekt ne?:) Minimalne bych ho tak ja nastavil a nechal bych jen klona bakalaru verejny.

Reply:
No to jsem taky našel, moc jsem to nezkoumal, ale nepodařilo se mi najít na to jak to nastavit tak jsem chtěl že tady by to šlo i vidět jak nastavit